app/src/main/res/values/strings.xml

```diff
@@ -185,100 +185,100 @@
     <string name="ui_theme">UI theme (app restart required)</string>
     <string name="theme_light">Light</string>
     <string name="theme_dark">Dark</string>
-    <string name="dont_show_boost">Don\'t show boost</string>
-    <string name="dont_show_reply">Don\'t show reply</string>
+    <string name="dont_show_boost">Don\'t show boosts</string>
+    <string name="dont_show_reply">Don\'t show replies</string>
     <string name="regex_filter">Regex filter</string>
     <string name="regex_error">Regex error</string>
     <string name="notification_delete">Empty notifications</string>
-    <string name="confirm_delete_notification">This will empty your notification on your instance.\nAre you sure?</string>
+    <string name="confirm_delete_notification">Empty all notifications on instance?</string>
     <string name="search_for">Search %1$s</string>
-    <string name="spoil_visibility_for_account">Visibility is spoiled for account.</string>
+    <string name="spoil_visibility_for_account">The account is visible.</string>
     <string name="contributor">contributor</string>
     <string name="thanks_for">Thanks for %1$s</string>
-    <string name="simple_list">Simple list(app restart required)</string>
-    <string name="mute_app_desc">Swipe to delete. You may need reload to show status published by unmuted apps.</string>
+    <string name="simple_list">Simple list (app restart required)</string>
+    <string name="mute_app_desc">Swipe → delete. Reload to show statuses published by unmuted apps.</string>

```

Comment:
on google's image  search result of "swipe right", most case it indicated "hold finger on screen and move it to right".
the action to delete from muted app requires "swipe left." means inverted.
Also, it is better not omit "to" in this context.

Reply:
There are a few others that now are just "swipe", and the user doesn't know if that means left or right.